modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientCodegen.java

```diff
@@ -948,6 +949,18 @@ private String toExampleValueRecursive(String modelName, Schema schema, Object o
         if (objExample != null) {
             example = objExample.toString();
         }
+
+        // check for recursive model
+        if (processedModels.contains(modelName)) {
+            // have already processed this model
+            // this happens when you have recursive or circular models
+            return fullPrefix + modelName + closeChars;
+        } else if(modelName != null && !modelName.isEmpty()){
+            // if the model is non-null and non-empty, add it to the set
+            processedModels.add(modelName);
```

Comment:
Shouldn't we only add a model when we have finished generating its sample?
What are our recursive base cases here for cycles?

Reply:
I'll go back and take a look at this again, thanks.  I didn't realize that the 'ran /bin/utils/ensure-up-to-date' was required and didn't pick up on the fact that my examples had changed that.

Reply:
@spacether Got a CircleCI build error that I'm having  a hard time interpreting.  Do you mind taking a look and providing pointers for how to reproduce locally?

Reply:
That error is unrelated. We. can ignore it.